Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Image for building packages in docker
+FROM golang:1.13-alpine as builder
+
+RUN apk --no-cache add ruby ruby-dev ruby-etc make gcc g++ rpm git tar && \
+    gem install --no-user-install --no-document fpm
+
+
+
+FROM builder AS build
```

Comment:
this `FROM` is useless. you can just remove it

Reply:
It's used to build image with artefacts and builder independently. And only builder is pushed to docker hub